src/test/resources/CsvParserTest/authorconfig_commasAndDoubleQuotes_test.csv

```diff
@@ -0,0 +1,4 @@
+Repository's Location,Branch,Author's GitHub ID,Author's Email,Author's Display Name,Author's Git Author Name,Ignore Glob List
+https://github.com/reposense/testrepo-Beta.git,master,‘Processed�‘Cooked�,,"Tay Fan Gao, Douya","Tay Fan Gao, Douya ""SOC, Y2S1""",
```

Comment:
the special symbols in the other test should not be included here: `‘Processed�‘Cooked�`